package.json

```diff
@@ -19,6 +19,8 @@
     "deploy:docs": "gulp deploy:docs",
     "lint": "tslint -t stylish -p .",
     "lint:fix": "yarn lint --fix",
+    "eslint": "eslint .",
```

Comment:
I would prefer to name this just `lint`. The fact that we use eslint for that is an implementation detail I don't care about.

Reply:
Just FYI, while the PR was in `WIP` state, I had both so I could run parallel tests of the two.  Certainly agree on `lint` 👍